scripts/src/internal/jwt.ts

```diff
@@ -0,0 +1,47 @@
+import * as moment from "moment";
+import { readFileSync } from "fs";
+import * as jsonwebtoken from "jsonwebtoken";
+
+import { getConfig } from "./config";
+
+const CONFIG = getConfig();
+
+type KeyMap = Map<string, { algorithm: string; key: Buffer; }> & {
+	random(): { id: string; key: Buffer; algorithm: string; };
+};
+const KEYS = (() => {
+	const map = new Map<string, { algorithm: string; key: Buffer; }>() as KeyMap;
+	map.random = function(this: Map<string, { algorithm: string; key: Buffer; }>) {
+		const entries = Array.from(this.entries()).map(([id, key]) => ({
+			id,
+			key: key.key,
+			algorithm: key.algorithm
+		}));
+
+		return entries[Math.floor(Math.random() * entries.length)];
+	};
+
+	return map;
+})();
```

Comment:
I'd prefer just making `KeyMap` a class with a function, `getRandomKey()`. I had to read through this file a couple of times until I got what's going on here

Reply:
fair enough